snap/hooks/configure

```diff
@@ -141,7 +141,7 @@ for key in $ALL_SERVICES; do
                 handle_svc "kong-daemon" "off"
                 handle_svc "security-proxy-setup" "off" 
                 # turn off security-secrets-setup too
-                handle_svc "security-secrets-setup" "off" 
+                #handle_svc "security-secrets-setup" "off" 
```

Comment:
Same comment as the install hook, I'd prefer you just remove this...

Reply:
sure, will remove it.